src/StakingNode.sol

```diff
@@ -192,27 +222,50 @@ contract StakingNode is IStakingNode, StakingNodeEvents, ReentrancyGuardUpgradea
         );
 
         for (uint256 i = 0; i < validatorIndices.length; i++) {
+            // If the validator is already exited, the effectiveBalanceGwei is 0.
+            // if the validator has not been exited, the effectiveBalanceGwei is whatever is staked
+            // (32ETH in the absence of slasing, and less than that if slashed)
             uint256 effectiveBalanceGwei = validatorFields[i].getEffectiveBalanceGwei();
+
             emit ValidatorRestaked(validatorIndices[i], oracleTimestamp, effectiveBalanceGwei);
```

Comment:
wouldn't you say that emitting this event is confusing if it turns out the validator already exited (i.e. `effectiveBalanceGwei == zero`). And therefore we should move this event inside the below "if". (i might be wrong).